draft-ietf-scitt-architecture.md

```diff
@@ -436,7 +441,7 @@ Each implementation of a Transparency Service MAY support additional metadata, s
 The role of Transparency Service can be decomposed into several major functions.
 The most important is maintaining an Append-only Log, the verifiable data structure that records Signed Statements, and enforcing a Registration Policy.
 It also maintains a service key, which is used to endorse the state of the Append-only Log in Receipts.
-All Transparency Services MUST expose standard endpoints for Registration of Signed Statements and Receipt issuance, which is described in TODO: scrapi.
+All Transparency Services MUST expose standard endpoints for Registration of Signed Statements and Receipt issuance, which is described in TODO: SCRAPI.
```

Comment:
```suggestion
```

Reply:
Lets not make SCRAPI a normative dependency of the architecture.

Reply:
+1, this gets cleaned up with #149